python/tvm/relay/op/transform.py

```diff
@@ -1320,3 +1320,55 @@ def adv_index(inputs):
         Output tensor.
     """
     return _make.adv_index(Tuple(inputs))
+
+
+def sparse_segment_sum(data, indices, segment_ids, num_segments=None):
+    """
+    Compute the sparse segment sum on the indices over the segment_ids
```

Comment:
I think we need a lot more documentation here. Please include a description of what a segment sum is and how a sparse segment sum is different from a dense one.

Reply:
I will include a link to the TF documentation which will make it clearer.